# port: /dev/port3
port: /dev/ttyACM0
baud: 57600
timeout: 0.1

# rate: Tan so chung cua arduino node
# rate: 50
rate:  25

base_frame: base_link

type: GP2Y0A21YK0F
angles: [-45, -25, -10, 0, 10, 25, 45, 155, 180, 205] # Goc bo tri senso, donvi: do
pins: [0,1,2,3,4,5,6,8,9,10] # Arduino Mega
sensor_rate: 25
result_rate: 2
